pkg/scheduler/scheduler.go

```diff
@@ -440,18 +440,19 @@ func (sched *Scheduler) scheduleOne() {
 	// Synchronously attempt to find a fit for the pod.
 	start := time.Now()
 	suggestedHost, err := sched.schedule(pod)
-	metrics.SchedulingAlgorithmLatency.Observe(metrics.SinceInMicroseconds(start))
```

Comment:
Also, in anyway, do not delete the existing metrics. You are expected to add more.

Reply:
This added at https://github.com/kubernetes/kubernetes/pull/58192/files#diff-0260dd7e154d411be493ada0753dc547R455 to reflect true time taken.

Reply:
I see, thanks for pointing out!